po/cs.po

```diff
@@ -1115,7 +1115,7 @@ msgstr "Další..."
 #: ../virtaal/plugins/spellchecker.py:361
 #, python-format
 msgid "Add \"%s\" to Dictionary"
-msgstr "Přidat „%s“ do slovníku"
+msgstr "Přidat \"%s\" do slovníku"
```

Comment:
Quoting styles are language specific.  This looks perfectly correct for `cs`.  Likely you ran pofilter with specifying a language as there are language specific adaptation.